library/psa_crypto_rsa.c

```diff
@@ -274,7 +274,10 @@ static psa_status_t rsa_export_public_key(
 #endif /* defined(BUILTIN_KEY_TYPE_RSA_KEY_PAIR) ||
         * defined(BUILTIN_KEY_TYPE_RSA_PUBLIC_KEY) */
 
-#if defined(BUILTIN_KEY_TYPE_RSA_KEY_PAIR)
+/* XXX Does MBEDTLS_PSA_ACCEL_KEY_TYPE_RSA_KEY_PAIR use these functions? Both
+ * of them or just rsa_generate_key? */
```

Comment:
Currently only `rsa_generate_key`.

Reply:
Does it use this implementation of `rsa_generate_key()` or is another `rsa_generate_key()` substituted in at link time?

Reply:
We don't do link-time substitution.